src/System.Security.Cryptography.Cng/src/Internal/Cryptography/Helpers.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Security.Cryptography;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+using ErrorCode = Interop.NCrypt.ErrorCode;
+
+namespace Internal.Cryptography
+{
+    internal static class Helpers
+    {
+        public static byte[] CloneByteArray(this byte[] src)
```

Comment:
CloneByteArray() is the one that's really common across the crypto libraries, IIRC. By itself, it doesn't merit creating a source coupling between these libraries.